grabbit/src/main/groovy/com/twcable/grabbit/client/servlets/GrabbitJobServlet.groovy

```diff
@@ -97,14 +97,14 @@ class GrabbitJobServlet extends SlingAllMethodsServlet {
                 else {
                     log.warn "Request was made to get the status of a job, but a job could not be found for the ID provided for ${request.pathInfo}."
                     response.setStatus(SC_BAD_REQUEST)
-                    response.writer.write("Request was made to get the status of a job, but a job could not be found for the ID provided for ${request.pathInfo}.")
+                    response.writer.write("Request was made to get the status of a job, but a job could not be found for the ID provided for ${request.pathInfo}. Make sure the job ID provided is numeric, or 'all'")
```

Comment:
At this point, it's guaranteed that the job ID is numeric, or 'all', by line :78. This case happens when a job ID is provided, but it legitimately does not map to a valid job.